internal/process/deprovisioning/remove_instance_step.go

```diff
@@ -57,6 +57,8 @@ func (s *RemoveInstanceStep) Run(operation internal.Operation, log logrus.FieldL
 		log.Info("Removing the RuntimeID field from the operation")
 		operation, backoff, _ = s.operationManager.UpdateOperation(operation, func(operation *internal.Operation) {
 			operation.RuntimeID = ""
+			operation.GardenerClusterName = ""
```

Comment:
Why not to do it in [handlers.go ](https://github.com/kyma-project/kyma-environment-broker/blob/eb0393a7df82aa60803126f5d704edad89b4448c/internal/suspension/handler.go#L145)? It is more suitable because that is were new operation is initialised. The current approach means that we are changing something for the process that resides in other places which might be harder to understand.